src/generic_search.cpp

```diff
@@ -61,6 +61,11 @@ std::chrono::milliseconds GetTuningTimeMax()
     return std::chrono::milliseconds{env::value(MIOPEN_TUNING_TIME_MS_MAX)};
 }
 
+std::size_t GetTuningPatience()
+{
+    return env::value(MIOPEN_TUNING_PATIENCE);
+}
+
```

Comment:
I'm not quite sure that we need this function. Since we need this "Patience" parameter right inside the one particular method of the one particular class, we can use the environmental variable just in-place, like
```diff
- std::size_t patience = GetTuningPatience();
+ std::size_t patience = env::value(MIOPEN_TUNING_PATIENCE);
```

Reply:
@cderb while you adapt that change, please run a clang-format to clear the current CI issue.